CosmosToRedis/CosmosToRediscs.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
```

Comment:
It seems like this is duplicate of `CosmosToRedis/CosmosToRedis.cs` please remove it if so.